README.md

```diff
@@ -1,2 +1,32 @@
 # json2html
 This is a repository to build HTML based in json struct
+
+## Base Struct
+The json component is basicly:
```

Comment:
`The json payload is as follow:`, ou pelo menos corrigir `basicly` pra `basically`